php/class-ee-site.php

```diff
@@ -1,13 +1,15 @@
 <?php
 
 use \Symfony\Component\Filesystem\Filesystem;
+use \EE\Model\Site;
 
 /**
  * Base class for Site command
  *
  * @package ee
  */
-abstract class EE_Site_Command {
+abstract class EE_Site_Command
+{
```

Comment:
why this bracket is again in new line? Please check it all places and correct it.